ParseException when fetching transaction date for VolvoFinans

A ParseException is thrown during parsing of the date string for VolvoFinans.

`Wed Feb 18 00:00:00 CET 2015` 

Apparently the parser has problem with the TimeZone. 

```java
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.liato.bankdroid/com.liato.bankdroid.TransactionsActivity}: java.lang.NumberFormatException: Invalid int: "Wed Feb 18 00:00:00 CET 2015"
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2693)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2760)
at android.app.ActivityThread.access$900(ActivityThread.java:177)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1448)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:145)
at android.app.ActivityThread.main(ActivityThread.java:5944)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1389)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1184)
Caused by: java.lang.NumberFormatException: Invalid int: "Wed Feb 18 00:00:00 CET 2015"
at java.lang.Integer.invalidInt(Integer.java:138)
at java.lang.Integer.parse(Integer.java:410)
at java.lang.Integer.parseInt(Integer.java:367)
at java.lang.Integer.parseInt(Integer.java:334)
at com.liato.bankdroid.banking.Transaction.compareTo(Transaction.java:72)
at com.liato.bankdroid.banking.Transaction.compareTo(Transaction.java:21)
at java.util.ComparableTimSort.countRunAndMakeAscending(ComparableTimSort.java:286)
at java.util.ComparableTimSort.sort(ComparableTimSort.java:153)
at java.util.ComparableTimSort.sort(ComparableTimSort.java:142)
at java.util.Arrays.sort(Arrays.java:1957)
at java.util.Collections.sort(Collections.java:1864)
at com.liato.bankdroid.TransactionsActivity.onCreate(TransactionsActivity.java:61)
at android.app.Activity.performCreate(Activity.java:6289)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2646)
... 10 more
```

--- a/bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Volvofinans.java
+++ b/bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Volvofinans.java
@@ -61,7 +61,7 @@ public class Volvofinans extends Bank {
 	private static final int BANKTYPE_ID = IBankTypes.VOLVOFINANS;
     private static final int INPUT_TYPE_USERNAME = InputType.TYPE_CLASS_PHONE;
     private static final String INPUT_HINT_USERNAME = "ÅÅÅÅMMDDXXXX";
-    private static SimpleDateFormat DATE_PARSER = new SimpleDateFormat("EEE MMM d HH:mm:ss zzz yyyy");
+    private static SimpleDateFormat DATE_PARSER = new SimpleDateFormat("EEE MMM d HH:mm:ss zzz yyyy", Locale.UK);
     private static SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd", new Locale("sv_SE"));
     private HashMap<String, String> mAccountUrlMappings = new HashMap<String, String>();
     

